microraiden/microraiden/test/fixtures/http_client.py

```diff
@@ -17,4 +17,4 @@ def request_patched(self: DefaultHTTPClient, request: str):
 
     http_client = DefaultHTTPClient(client, api_endpoint, api_endpoint_port, retry_interval=0.5)
     http_client._request_resource = types.MethodType(request_patched, http_client)
-    return http_client
+    yield http_client
```

Comment:
Did this cause any problems?

Reply:
I was messing with the code while trying to figure out why client is kept alive after the test ends. `yield` and `return` should be equal, no harm here IMO